src/components/chart/chart.types.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Chart component props.
+ * @beta
+ */
+export interface ChartItem<
+    T extends number | [number, number] = number | [number, number],
+> {
+    /**
+     * label displayed for the item.
+     */
+    text: string;
+
+    /**
+     * value of the item.
+     */
+    value: T;
+
+    /**
+     * A prefix shown before the value
+     */
+    prefix?: string;
+
+    /**
+     * A suffix shown after the value.
+     */
+    suffix?: string;
```

Comment:
I think we should remove these from this interface and find a better way to format the values

Reply:
I don't know what you mean by "find a better way". Can you explain a bit more?

But your comments reminds me of what you did in the info tiles, using standard browser-formatting of values. Is that what you mean?

Also keep in mind that these are used to improve accessibility.

Reply:
It's better to let the consumer handle the formatting of the values, since we do not know anything about them. I added a `formattedValue` prop now instead that will be used if provided

Reply:
We could also have added some formatter function that the consumer could provide, but this seemed like the easiest approach right now

Reply:
first I thought it's like `limel-info-tile` where we have suffix and prefix. But I think the story is completely different there, since we render them in different places in the UI. So this approach is perhaps better. But on the other hand, it forces the consumer to also come up with a way to create their formatted value (which is OK)